vendor.conf

```diff
@@ -1,9 +1,9 @@
 github.com/urfave/cli v1.17.0
-github.com/containers/image master
+github.com/containers/image update_rename_logrus https://github.com/mrunalp/image
```

Comment:
@mrunalp can you put master back and revendor now that I've merged the PR in c/image?